### ISSUE TITLE
Unhandled rejection Error with multi/exec

### ISSUE THREAD

**Issue by goldfire** [luin/ioredis#313]
If I set `dropBufferSupport` to `true`, I get the following error when running `exec`:

```
Unhandled rejection Error: *Buffer methods are not available because "dropBufferSupport" option is enabled.Refer to https://github.com/luin/ioredis/wiki/Improve-Performance for more details.
    at Pipeline.<anonymous> (/path/to/node_modules/ioredis/lib/commander.js:118:31)
    at Pipeline.exec (/path/to/node_modules/ioredis/lib/pipeline.js:203:23)
    at Pipeline.pipeline.exec (/path/to/node_modules/ioredis/lib/transaction.js:32:14)
    at /path/to/game/lib/server/RedisStore.js:37:12
    at tryCatcher (/path/to/node_modules/ioredis/node_modules/bluebird/js/release/util.js:16:23)
    at Promise.successAdapter [as _fulfillmentHandler0] (/path/to/node_modules/ioredis/node_modules/bluebird/js/release/nodeify.js:23:30)
    at Promise._settlePromise (/path/to/node_modules/ioredis/node_modules/bluebird/js/release/promise.js:556:21)
    at Promise._settlePromise0 (/path/to/node_modules/ioredis/node_modules/bluebird/js/release/promise.js:604:10)
    at Promise._settlePromises (/path/to/node_modules/ioredis/node_modules/bluebird/js/release/promise.js:683:18)
    at Async._drainQueue (/path/to/node_modules/ioredis/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/path/to/node_modules/ioredis/node_modules/bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/path/to/node_modules/ioredis/node_modules/bluebird/js/release/async.js:17:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

No `buffer` methods are being called, so why am I seeing this error?

```javascript
redis.multi()
    .srem('set1', reply)
    .srem('set2', reply)
    .srem('set3', reply)
    .srem('set4', reply)
    .srem('set5', reply)
    .exec();
```

**Comment by luin** [comment]
It's a bug of ioredis v2.0.0. Should be fixed in v2.0.1. Thank you for pointing this out!

**Comment by goldfire** [comment]
Working now, thanks!
